src/pages/dar_application/DataUseAgreements_new.js

```diff
@@ -0,0 +1,81 @@
+        ]),
+
+        ol({ className: 'data-use-list' }, [
+          li({}, ['Use datasets solely in connection with the research project described in the approved Data Access Request for each dataset; ']),
+          li({}, ['Make no attempt to identify or contact individual participants or groups from whom data were collected, or generate information that could allow participants’ identities to be readily ascertained, without appropriate approvals from the submitting institutions;']),
+          li({}, ['Maintain the confidentiality of the data and not distribute them to any entity or individual beyond those specified in the approved Data Access Request;']),
+          li({}, ['Adhere to the NIH Security Best Practices for Controlled-Access Data Subject to the NIH Genomic Data Sharing Policy and ensure that only approved users can gain access to data files;']),
+          li({}, ['Acknowledge the Intellectual Property terms as specified in the Library Card Agreement; ']),
+          li({}, ['Provide appropriate acknowledgement in any dissemination of research findings including the investigator(s) who generated the data, the funding source, accession numbers of the dataset, and the data repository from which the data were accessed; and,']),
+          li({}, [
+            'Report any inadvertent data release, breach of data security, or other data management incidents in accordance with the terms specified in the ', a({target: '_blank', href: LibraryCardAgreementLink}, ['Library Card Agreement']), ' and ', a({target: '_blank', href: ''}, ['NIH Data Use Certification.'])]),
+        ])
+      ]),
+
+      h3({}, [
+        'By submitting this data access request, you agree to all terms in the agreement(s) listed below.'
+      ]),
+
+      div({ className: 'flex flex-row', style: { justifyContent: 'flex-start', }, }, [
+        div({}, [
+          a({ target: '_blank', href: LibraryCardAgreementLink, className: 'button button-white', style: { marginRight: '2rem', }, }, [
+            span({className: 'glyphicon glyphicon-download'}),
+            ' Broad Library Card Agreement'
+          ])
+        ]),
+        div({}, [
+          a({ target: '_blank', href: '', className: 'button button-white' }, [
+            span({className: 'glyphicon glyphicon-download'}),
+            ' NIH Library Card Agreement'
+          ])
+        ])
```

Comment:
Do we need a file for this download?

Reply:
yes, I asked pamela about the link and she said we are still waiting on it from NIH

Reply:
I think we have this file now in slack.